docs/reference/objects/company.md

```diff
@@ -51,6 +51,13 @@ string
 
 The Facebook username set within **My Site** > **Social**.
 
+## `company.id`
```

Comment:
Thank you for alphabetising ✨ 
Whilst we're touching this, can we move `booking_fee_name` and `organisation` into alphabetical order too please?